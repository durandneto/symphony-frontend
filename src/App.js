import React from 'react'
import {
  BrowserRouter as Router,
  Switch,
  Route,
} from "react-router-dom"

import { BundleAnalyzer } from './features/bundleAnalyzer/BundleAnalyzer'
import { HomePage } from './features/home/Home'

function App() {
  return (
    <Router>
      <Switch>
        <Route path="/:term" component={BundleAnalyzer} />
        <Route path="/" component={HomePage} />
      </Switch>
    </Router>
  )
}

export default App
