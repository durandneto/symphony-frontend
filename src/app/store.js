import { configureStore } from '@reduxjs/toolkit';
import BundleAnalyzerSlice from '../features/bundleAnalyzer/BundleAnalyzerSlice';

export default configureStore({
  reducer: {
    BundleAnalyzer: BundleAnalyzerSlice,
  },
});
