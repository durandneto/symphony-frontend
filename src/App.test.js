import React from 'react'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'
import store from './app/store'
import App from './App'

test('renders learn react link', () => {

  const result = renderer.create(
    <Provider store={store}>
      <App />
    </Provider>
  ).toJSON()
  expect(result).toMatchSnapshot()
})
