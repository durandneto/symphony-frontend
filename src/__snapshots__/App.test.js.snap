// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="sc-AxjAm hTSXUB"
>
  <div
    className="sc-AxjAm sc-AxirZ fssdFE"
  >
    <div
      className="sc-AxjAm sc-AxiKw jRLkDN"
    >
      <img
        alt="Symphony search"
        src="Symphony-logo.jpg"
        title="Symphony search"
      />
    </div>
  </div>
  <div
    className="sc-AxjAm sc-AxirZ fssdFE"
  >
    <div
      className="sc-AxjAm sc-AxiKw jRLkDN"
    >
      <input
        className="sc-fzoyAV fMTSuE"
        disabled={false}
        loading={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyUp={[Function]}
        placeholder="find a package"
        type="text"
      />
    </div>
  </div>
</div>
`;
